pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -67,12 +67,13 @@ def create_task(  # pylint: disable=too-many-statements,too-many-branches
 
         Args:
             **kwargs: Additional keyword arguments passed along with
-                             request and start_time.
+                             request.
```

Comment:
What's with the weird indentation? I know you haven't added it in this comment, but you might as well fix it. Usually, indentation in Google-style docstrings for functions & methods is like this:

```console
"""1-line summary; start with verb in imperative & don't exceed 80 chars

Optional extended description, wrapping when max line length of 80 chars
is exceeded. No indentation on successive lines.

Args: (same for "Attributes" & "Raises" sections)
    keyword: keyword is indented one level
    keyword_2: if the keyword description exceeds 80 chars, then
        indent any additional lines by _one_ more level.

Returns:
    Summary description of return value in one line.

    If more info is needed, add it to an additional line that is separated
    by one empty line. Just like in the extended function/method
    description, do not use extra indentation.

"""
```

[Here is more info](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html), including another detailed example:

```python
def module_level_function(param1, param2=None, *args, **kwargs):
    """This is an example of a module level function.

    Function parameters should be documented in the ``Args`` section. The name
    of each parameter is required. The type and description of each parameter
    is optional, but should be included if not obvious.

    If \*args or \*\*kwargs are accepted,
    they should be listed as ``*args`` and ``**kwargs``.

    The format for a parameter is::

        name (type): description
            The description may span multiple lines. Following
            lines should be indented. The "(type)" is optional.

            Multiple paragraphs are supported in parameter
            descriptions.

    Args:
        param1 (int): The first parameter.
        param2 (:obj:`str`, optional): The second parameter. Defaults to None.
            Second line of description should be indented.
        *args: Variable length argument list.
        **kwargs: Arbitrary keyword arguments.

    Returns:
        bool: True if successful, False otherwise.

        The return type is optional and may be specified at the beginning of
        the ``Returns`` section followed by a colon.

        The ``Returns`` section may span multiple lines and paragraphs.
        Following lines should be indented to match the first line.

        The ``Returns`` section supports any reStructuredText formatting,
        including literal blocks::

            {
                'param1': param1,
                'param2': param2
            }

    Raises:
        AttributeError: The ``Raises`` section is a list of all exceptions
            that are relevant to the interface.
        ValueError: If `param2` is equal to `param1`.

    """
```

Reply:
Will fix this indentation in the next commit, 
Moreover, there are issues in the explanation of the docstring of multiple functions in the codebase will create a separate issue for that and fix it there only.

Reply:
That makes sense, thanks a lot :)

Reply:
Done